src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -4,46 +4,62 @@
 
 package frc.robot.subsystems;
 
-
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.networktables.NetworkTable;
 import edu.wpi.first.networktables.NetworkTableEntry;
 import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Vision extends SubsystemBase {
   private NetworkTable table;
-  private NetworkTableEntry tx;
-  private NetworkTableEntry ty;
-  private NetworkTableEntry ta;
+  private NetworkTableEntry validTarget;
+  private NetworkTableEntry xOffset;
+  private NetworkTableEntry targetArea; // proxy for distance from target
+  private NetworkTableEntry targetID;
```

Comment:
I like the idea of all these target related variables of all having a consistent naming like targetArea, targetValid, targetXOffset, etc